api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ObservationValueTranslatorImpl.java

```diff
@@ -50,8 +50,9 @@ public Type toFhirResource(@Nonnull Obs obs) {
 		
 		// IMPORTANT boolean values are stored as a coded value, so for this to
 		// work, we must check for a boolean value before a general coded value
-		if (obs.getValueBoolean() != null) {
-			return new BooleanType(obs.getValueBoolean());
```

Comment:
This allows mocking out the `obs.getValueBoolean()` method in tests without having to introduce static mocks, which often cause issues dirtying the context.